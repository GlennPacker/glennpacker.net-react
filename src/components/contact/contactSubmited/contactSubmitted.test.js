import React from 'react';
import { shallow } from 'enzyme';

import ContactSubmited from './index';

describe('Contact Submitted', () => {
    let wrapper;

    beforeAll(() => {
        wrapper = shallow(
            <ContactSubmited />
        );
    })

    it('should render matching snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    })
})