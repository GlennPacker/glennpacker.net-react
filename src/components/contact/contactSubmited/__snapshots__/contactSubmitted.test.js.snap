// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Submitted should render matching snapshot 1`] = `
<div
  className="container pt-5"
>
  <Style
    rules={
      Object {
        ".contact-img": Object {
          "borderRadius": "50%",
          "height": 230,
          "width": 230,
        },
      }
    }
    scopeSelector=""
  />
  <div
    className="row"
  >
    <div
      className="col-sm"
    >
      <img
        alt="contact glenn"
        className="contact-img"
        src="GlennPacker.jpg"
      />
    </div>
    <div
      className="col-sm"
    >
      <h1>
        Thank you
      </h1>
      <p
        className="pt-3"
      >
        I will get back to you as soon as we can.
      </p>
    </div>
    <div
      className="col-sm"
    >
      <img
        alt="mailman waiting"
        className="contact-img"
        src="cat-waiting-for-the-mail.jpg"
      />
    </div>
  </div>
</div>
`;
